src/pyOpenMS/pxds/MRMDecoy.pxd

```diff
@@ -29,24 +29,24 @@ cdef extern from "<OpenMS/ANALYSIS/OPENSWATH/MRMDecoy.h>" namespace "OpenMS":
                             bool enable_unspecific_losses,
                             int round_decPow) nogil except +
             # wrap-doc:
-                #   Generate decoys from a TargetedExperiment
-                #   -----
+                #   Generate decoys from a TargetedExperiment\n
```

Comment:
Same here. If this is necessary everywhere, it needs to be fixed in autowrap.

Reply:
Ok I will remove it from here but we will need to add it in autowrap.